src/StoreApi/Schemas/CartCouponSchema.php

```diff
@@ -8,6 +8,7 @@
  *
  * @internal This API is used internally by Blocks--it is still in flux and may be subject to revisions.
  * @since 2.5.0
+ * @since $VID:$ Coupon type (`discount_type`) added.
```

Comment:
This API is not public yet so I don't think we need this.

Reply:
Even though it's not public I still think it's useful to track when fields are added or schema changed – doesn't hurt right?